### DISCUSSION TITLE
"Galaxy Brain" badge not found in my profile

### DISCUSSION THREAD

**Discussion by fayasnoushad** [community/community#18571] [Other Feature Feedback, Questions, & Ideas] [answered]
I answered 4 discussions, but "Galaxy Brain" achievement not found in my profile.

**Comment by Aadhi000** [comment]
**Bro You Will Only Get The Galaxy Brain badge after 2 Selected Answers**

**Comment by number0x0001** [comment, marked as answer]
You must have two answers and you cannot select your own comments as answers.

**Reply by fayasnoushad** [reply]
Ok
